client/app/views/templates/contacts/merge.jade

```diff
@@ -0,0 +1,89 @@
+extends ../../../lib/views/templates/base/dialog.jade
+
+
+block content
+    include helpers.jade
+
+    mixin radio(options, fieldName)
+      ul
+          - for (var i=0; i< options.length; i++)
+              -var option = options[i]
+                  li
+                      label
```

Comment:
`label` tag should come with a `for` attribute mapped to the attached input `id` value. Can you add it to keep it consistent accross the whole app? Thanks :smile: